### PR TITLE
NMS-12541: Bumped version for io.netty

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://issues.opennms.org/browse/NMS-12541

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1227,7 +1227,7 @@
     <ecjVersion>4.4.2</ecjVersion>
     <eclipseGeminiVersion>2.0.0.RELEASE</eclipseGeminiVersion>
     <eclipselinkVersion>2.5.1</eclipselinkVersion>
-    <elasticsearchNettyVersion>4.1.30.Final</elasticsearchNettyVersion>
+    <elasticsearchNettyVersion>4.1.48.Final</elasticsearchNettyVersion>
```
Comment (j-white) [inline pom.xml:1230]: This one is only used for testing and should match the version that Elasticsearch is using - no need to update it.

**Comment by RangerRick** [comment]
Rebased now that `release-26.1.0` exists.  Though, is this a thing that should happen in `foundation-2019` too?  Or would that require other changes?

**Comment by christianpape** [comment]
I'll check this for `foundation-2019`.

**Comment by christianpape** [comment]
Opened another PR https://github.com/OpenNMS/opennms/pull/2974 for the change in foundation-2019. I'll merge this one too, if Bamboo is green.

**Pull request merged**